src/js/views/boardview/boardview.js

```diff
@@ -3,15 +3,23 @@ import classNames from 'classnames';
 import Card from '../cardview/cardview';
 import Avatar from '../avatarview/avatarview';
 import Button from '../buttonview/buttonview';
+import Bullet from "../bullet/bullet";
+import Client from "../../client"
+import GameState from "../../gamestate"
 import './_boardview.scss';
 
 export default class GameBoard extends Component {
 
-  resign() {
-    alert('You have resigned!');
-    window.client.makeLocalMove({"op":"resign"});
+  returnToLobby() {
+    window.client.makeLocalMove({"op":"resign"})
```

Comment:
This should be done with a router instead. Not sure what `rootPointer` is doing.